casino_classes.py

```diff
@@ -0,0 +1,61 @@
+cardsInDeck = [suit + str(val) for suit in 'cdhs' for val in [i for i in range(2, 11)] + ['j', 'q', 'k', 'a']]
```

Comment:
caps on the letters please (dict keys are "SA" for ace of spades)

Reply:
got it